core/gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -281,14 +287,33 @@ export class ResultPanelComponent implements OnInit, OnDestroy {
 
   resetPanelPosition() {
     this.dragPosition = { x: this.returnPosition.x, y: this.returnPosition.y };
+    this.isDocked = true;
+  }
+
+  checkPanelIsDocked() {
+    this.isDocked = this.returnPosition.x === this.dragPosition.x && this.returnPosition.y === this.dragPosition.y;
+  }
+
+  getResizeDirections() {
```

Comment:
here replace returning variable with return constant array

Reply:
changed; I had to put the if-else logic directly into the HTML file because the panels became unresizable when I tried to return the constant arrays. I spent time trying to debug it but couldn't fix it and had to put it in HTML directly